std/boolean.morganey

```diff
@@ -1,2 +1,2 @@
-true := (\a . (\b . a))
+true := \a.b.a
```

Comment:
Is there a particular reason you're using `\` here and `λ` in `arithmetic.morganey`? If no, then I think it's better to be consistent among all the standard library.